docs/UserGuide.md

```diff
@@ -224,6 +234,7 @@ _Details coming soon ..._
 6. Edit command only supports editing user's personal information (name, phone number, email, address and tags, but not merit score and borrowed books).
 7. Add command is used to record new user's personal information into the User List.
 8. Add and Edit Command **DOES NOT** support adding/editing merit score or borrowed book.
+9. Only Borrow, Donate and Return command is for managing books.
```

Comment:
Some nit picks, I feel like changing
... Return *commands are is more grammatically accurate.